### PR TITLE
backport increase block gas limit

### PR DESCRIPTION
**Pull request by thiolliere**
I don't know if v1.0 status but, maybe cool to add it as well

### CODE REVIEW

**Comment by thiolliere** [comment]
related to https://github.com/paritytech/substrate/pull/2597
and https://github.com/paritytech/substrate/issues/2566

**Comment by xlc** [comment]
This have no effects on existing chain so why need to bump version?

**Comment by bkchr** [comment]
Good point @xlc. Next question would be, as `ink!` targets master, do we need this in 1.0?

**Comment by xlc** [comment]
I see no downside to increase it. Any public chain have to override default values in chain spec to match their economic / security model so this only really applies to dev testnet, and a default value that is too high is better one that is too low cause otherwise contracts may be rejected unexpectedly.

File: node/runtime/src/lib.rs [hunk 1]
```diff
@@ -59,7 +59,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
 	spec_version: 60,
-	impl_version: 60,
+	impl_version: 61,
```
Comment (thiolliere) [inline node/runtime/src/lib.rs]:
```suggestion
	impl_version: 60,
```

**Pull request merged**